src/app/components/grid/index.html

```diff
@@ -128,4 +170,227 @@
+			<p>If the user is likely to need to sort based on the data in a column, you can enable sorting so that the user can click the header of the column to toggle between sorting ascending and descending for the specified field in that column.</p>
+		</stache-column>
+		<stache-column screenSmall="6">
+			<div class="screenshot screenshot-info">
+				<img src="~/assets/img/guidelines/grid/grid-option-sort.png" style="width: 100%;" />
+			</div>
+		</stache-column>
+	</stache-row>
+    
+	<header><div class="sky-subsection-heading">Templated columns</div></header>
+
+  <stache-row>
+    <stache-column screenSmall="6">
+			<p>Some columns may need to represent composite information rather than a single value. You can use templated columns to lay the content out as needed.</p>
+		</stache-column>
+		<stache-column screenSmall="6">
+			<div class="screenshot screenshot-info">
+				<img src="~/assets/img/guidelines/grid/grid-option-template-column.png" style="width: 100%;" />
+			</div>
+		</stache-column>
+	</stache-row>
+    
+	<header><div class="sky-section-heading">Behavior</div></header>
+
+	<header><div class="sky-subsection-heading">Fit</div></header>
+	
+	<header><div class="sky-emphasized">Scroll</div></header>	
+		
+	
+		<stache-row>
+			<stache-column screenSmall="6">
+				<p>Setting fit to “scroll” lets the grid take as much space as needed by adding a horizontal scrollbar. This is most useful for grids with large numbers of columns.</p>
+			</stache-column>
+			<stache-column screenSmall="6">
+			<div class="screenshot screenshot-info">
+				<img src="~/assets/img/guidelines/grid/grid-fit-scroll.png" style="width: 100%;" />
+			</div>
+		</stache-column>
+		</stache-row>	
+		
+	
+	<header><div class="sky-emphasized">Width</div></header>
+  
+	
+		<stache-row>
+			<stache-column screenSmall="6">
+				<p>Setting fit to “width” fixes the width of the grid to that of its container. Columns resize themselves to fit. This is most useful for simple grids in smaller containers such as tiles, where the content will fit neatly in the allotted space.</p>
+			</stache-column>
+			<stache-column screenSmall="6">
+			<div class="screenshot screenshot-info">
+				<img src="~/assets/img/guidelines/grid/grid-fit-width.png" style="width: 100%;" />
+			</div>
+		</stache-column>
+		</stache-row>	
+		
+	
+	<header><div class="sky-subsection-heading">Reordering columns</div></header>
+
+  <stache-row>
+    <stache-column screenSmall="6">
+			<p>The user can reorder columns via drag and drop.</p>
+		</stache-column>
+		<stache-column screenSmall="6">
+			<div class="screenshot screenshot-success">
+				<img src="~/assets/img/guidelines/grid/grid-reorder.png" style="width: 100%;" />
+			</div>
+		</stache-column>
+	</stache-row>
+    
+	<header><div class="sky-subsection-heading">Resizing columns</div></header>
+
+  <stache-row>
+    <stache-column screenSmall="6">
+			<p>The user can resize columns via drag and drop or the keyboard.</p>
```

Comment:
"Drag and drop" seems fitting when talking about reordering columns. However, is that the right term when referring to resizing? Would something like "drag to resize" be more fitting?

Reply:
I agree that this seems like the wrong term. I think this should be something like "The user can resize columns by clicking and dragging or by using the keyboard." Or maybe "The user can resize columns using the mouse or the keyboard." ... @Blackbaud-ToddRoberts, what do you think?

Reply:
@blackbaud-johnly your first option sounds good to me